File: weekly_test11/devowel_file.py
#!/usr/bin/python3

import sys
import re

with open(sys.argv[1], "r+") as file:

	line = file.read()

line = re.sub("[AaEeIiOoUu]", "", line)

file.close()
with open(sys.argv[1], "w") as file:
	file.write(line)

File: weekly_test11/devowel_file.pl
#!/usr/bin/perl -w

$file = $ARGV[0];

open F, '<', $file or die "can't open file";
@oldfile = <F>;
close(F);

foreach (@oldfile){
	s/[aeiou]//ig;
}

open FILE, ">", $file or die "$!";
print FILE @oldfile;
close(FILE);